### PR TITLE
library: add radosgw modules

### PR DESCRIPTION
**Pull request by dsavineau**
This adds multiple radosgw modules for managing radosgw user, realm, zonegroup and zone ressources.

- radosgw_user
- radosgw_realm
- radosgw_zonegroup
- radosgw_zone

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

File: tests/library/test_radosgw_user.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import json
```
Comment (guits) [inline tests/library/test_radosgw_user.py:1]: credit toi maggle

File: library/radosgw_user.py [hunk 2]
```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/python3
```
Comment (guits) [inline library/radosgw_user.py:1]: credit toi maggle

Comment (dsavineau) [reply library/radosgw_user.py:1]: We don't do that in the other modules

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-all_daemons

**Review by guits — APPROVED** [review]

**Pull request merged**